python/ray/train/examples/pytorch/torch_fashion_mnist_example.py

```diff
@@ -48,111 +61,81 @@ def forward(self, x):
         return logits
 
 
-def train_epoch(dataloader, model, loss_fn, optimizer):
-    size = len(dataloader.dataset) // train.get_context().get_world_size()
-    model.train()
-    for batch, (X, y) in enumerate(dataloader):
-        # Compute prediction error
-        pred = model(X)
-        loss = loss_fn(pred, y)
+def train_func_per_worker(config: Dict):
```

Comment:
FYI this is imported in a few tests which may need to be updated?

Reply:
Done! I think adding the "per_worker" suffix would be more clear to users, so I updated it here:)